### ISSUE TITLE
Missing LMU entries in Django Managed Areas

### ISSUE THREAD

**Issue by kkemp85** [legiongis/hmsf-md#126]
A number of LMU entries are missing from the "Managed Area" section on the Django site. Missing entries include LMUs from State Parks, Forestry, and FWC managed lands. All LMUs missing from State Parks and forests already have Arches accounts created and have entries in the RDM. Some of the missing FWC entries have Arches accounts and exist in the RDM and some do not. I will work on creating new accounts and adding entries into the RDM for those missing there. I can also provide lists of these missing LMUs and their polygons as well.

**Comment by mradamcox** [comment]
@kkemp85 and I figured out that the missing land management areas, _are_ in the fixtures that are loaded into Arches when the database is first created, but for some reason their loading failed (no errors were reported though). Solving this will require a reevaluation of the load fixtures themselves to compare them with what actually gets saved in the database.

I'm fairly certain this is the underlying cause of #64 and #61 (which really could be combined, but I think we may as well wait until this issue is addressed to make sure).

**Comment by mradamcox** [comment]
@kkemp85 I think we can probably close this issue at this point, can you confirm?

**Comment by kkemp85** [comment]
@mradamcox Yes, we can. They aren't all in there, but I am handling adding anything that is missing from now on, so it is no longer an issue for you.
